File: 070/cpp/main.cpp
class Solution {
public:
    int climbStairs(int n) {
        if (n == 1) {
            return 1;
        }
        int a = 1;
        int b = 1;
        for (int i = 1; i < n; i++) {
            int t = a + b;
            a = b;
            b = t;
        }
        return b;
    }
};

File: 070/rust/main.rs
impl Solution {
    pub fn climb_stairs(n: i32) -> i32 {
        if n == 1 {
            return 1;
        }
        let mut a = 1;
        let mut b = 1;
        for _ in 1..n {
            let t = a + b;
            a = b;
            b = t;
        }
        return b;
    }
}
